contracts/DCAStrategies/DCAStrategies/DCAStrategiesPermissionsHandler.sol

```diff
@@ -71,4 +81,39 @@ abstract contract DCAStrategiesPermissionsHandler is IDCAStrategiesPermissionsHa
   /// @inheritdoc IDCAStrategiesPermissionsHandler
   // TODO: update this after building the new descriptor
   function setNFTDescriptor(IDCAHubPositionDescriptor _descriptor) external override {}
+
+  function _mint(
+    uint256 _id,
+    address _owner,
+    IDCAStrategies.PermissionSet[] calldata _permissions
+  ) internal {
+    _mint(_owner, _id);
+    _setPermissions(_id, _permissions);
+  }
+
+  // not sure about the name of this fn
+  function __burn(uint256 _id) internal {
```

Comment:
Maybe we can call it `_burn` and override the ERC721 version. Is it overridable?